src/drivers/ports/avr/uart_port.h

```diff
@@ -33,10 +33,32 @@
 
 #include <io.h>
 
-/*
- * UART modes. See https://en.wikipedia.org/wiki/Serial_port
- */
-#define UART_PORT_FRAME_FORMAT_DEFAULT 0
+#define UART_PORT_FRAME_FORMAT_5N1 0x00
+#define UART_PORT_FRAME_FORMAT_6N1 0x02
+#define UART_PORT_FRAME_FORMAT_7N1 0x04
+#define UART_PORT_FRAME_FORMAT_8N1 0x06
+#define UART_PORT_FRAME_FORMAT_5N2 0x08
+#define UART_PORT_FRAME_FORMAT_6N2 0x0A
+#define UART_PORT_FRAME_FORMAT_7N2 0x0C
+#define UART_PORT_FRAME_FORMAT_8N2 0x0E
+#define UART_PORT_FRAME_FORMAT_5E1 0x20
+#define UART_PORT_FRAME_FORMAT_6E1 0x22
+#define UART_PORT_FRAME_FORMAT_7E1 0x24
+#define UART_PORT_FRAME_FORMAT_8E1 0x26
+#define UART_PORT_FRAME_FORMAT_5E2 0x28
+#define UART_PORT_FRAME_FORMAT_6E2 0x2A
+#define UART_PORT_FRAME_FORMAT_7E2 0x2C
+#define UART_PORT_FRAME_FORMAT_8E2 0x2E
+#define UART_PORT_FRAME_FORMAT_5O1 0x30
+#define UART_PORT_FRAME_FORMAT_6O1 0x32
+#define UART_PORT_FRAME_FORMAT_7O1 0x34
+#define UART_PORT_FRAME_FORMAT_8O1 0x36
+#define UART_PORT_FRAME_FORMAT_5O2 0x38
+#define UART_PORT_FRAME_FORMAT_6O2 0x3A
+#define UART_PORT_FRAME_FORMAT_7O2 0x3C
+#define UART_PORT_FRAME_FORMAT_8O2 0x3E
```

Comment:
Use the defines `UCSZxx`, `UPMxx` and `USBSx`, as in `(_BV(UCSZ00) | _BV(UCSZ01))`.